execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -331,7 +329,7 @@ impl TransactionGenerator {
     }
 
     /// Verifies the sequence numbers in storage match what we have locally.
-    pub fn verify_sequence_number(&self, db: &dyn DbReader) {
+    pub fn verify_sequence_number(&self, db: Arc<dyn DbReader>) {
```

Comment:
any particular reason for this change?

Reply:
The call site already has Arc<dyn DbReader>, which also needs to be passed to the `get_state_value_resolver_for_latest_version` API